install_config/configuring_sdn.adoc

```diff
@@ -212,37 +212,41 @@ $ oc get netnamespace
 ----
 
 [[migrating-between-sdn-plugins-networkpolicy]]
-=== Migrating from the ovs-multitenant Plugin to ovs-networkpolicy
+=== Migrating from ovs-multitenant to ovs-networkpolicy
 
-Before migrating from *ovs-multitenant* to *ovs-networkpolicy*, you must ensure that every
-namespace has a unique `NetID`; that means that if you have previously
+Before migrating from the *ovs-multitenant* plugin to the *ovs-networkpolicy*
+plugin, ensure that every namespace has a unique `NetID`. This means that if you
+have previously
 xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined projects
 together] or
 xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made projects
-global], you will need to undo that before switching to the *ovs-networkpolicy* plugin, or
-NetworkPolicy objects may not function correctly.
+global], you will need to undo that before switching to the *ovs-networkpolicy* plugin,
+or the NetworkPolicy objects may not function correctly.
 
-A helper script is available that fixes `NetID`s, creates NetworkPolicy objects to isolate
-previously-isolated namespaces, and enabled connections between previously-joined
-namespaces.
+A helper script is available that fixes `NetID's`, creates NetworkPolicy objects
+to isolate previously-isolated namespaces, and enables connections between
+previously-joined namespaces.
 
-. Download https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh
+Use the following steps to migrate to the *ovs-networkpolicy*
+plugin, by using this helper script, while still running the *ovs-multitenant* plugin:
+
+. Download the script from link:https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh[] and add the exexcution file permission:
 +
+[source, bash]
 ----
 $ curl https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh
 $ chmod a+x migrate-network-policy.sh
 ----
-. While still running the *ovs-multitenant* plugin (that is, before running any of the
-migration steps above), run that script as a user with cluster administrator rights on
-the OpenShift cluster
+. Run the script (requires Cluster Administration role).
 +
+[source, bash]
 ----
 $ ./migrate-network-policy.sh
 ----
 
-After running the script, every project will now be fully isolated from every other
-project, so connection attempts between pods in different projects will fail until you
-complete the migration to the *ovs-networkpolicy* plugin.
+After running this script, every project is fully isolated from every other
+project, therefore connection attempts between pods in different projects will
+fail until you complete the migration to the *ovs-networkpolicy* plugin.
```

Comment:
This is an issue throughout the docs in general, I think. Should we be consistent with project/namespace?  Before the script examples, you used namespace.